webapp/channels/src/components/custom_status/custom_status_suggestion.tsx

```diff
@@ -42,25 +41,19 @@ const CustomStatusSuggestion: React.FC<Props> = (props: Props) => {
     };
 
     const clearButton = handleClear ? (
-        <div
-            className='suggestion-clear'
-        >
-            <OverlayTrigger
-                delayShow={Constants.OVERLAY_TIME_DELAY}
+        <div className='suggestion-clear'>
+            <WithTooltip
                 placement='top'
-                overlay={
-                    <Tooltip id='clear-recent-custom-status'>
-                        {'Clear'}
-                    </Tooltip>
-                }
+                title='Clear'
```

Comment:
Just a bit more of an ask : i noticed that we didnt translate this word, can you use format intl hook to add translation here?

Reply:
Sure, I've added the translation for 'Clear' using FormattedMessage.